app/main.py

```diff
@@ -1 +1,35 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    parts = command.split()
```

Comment:
Consider using unpacking for the `parts` list to improve code readability and efficiency. According to the checklist, it's better to use unpacking when assigning values from a list to individual variables. For example: `command, source, destination = parts`.